README.md

```diff
@@ -30,50 +29,47 @@ $ pip install meeshkan
 
 Note that `meeshkan` requires **Python 3.6+.**
 
-## Python API
+## Hello world
 
-### Quick start
+The basic Meeshkan flow is **collect, build and mock.**
+1. To start, **collect** data from recorded server traffic and/or OpenAPI specs.
+1. Then, **build** a schema that unifies these various data sources.
+1. Finally, use this schema to create a **mock** server of an API.
 
```

Comment:
A "Hello world" type article should help guide the user through their first experience using the product. So this section needs a bit more context about what we expect the user to do exactly, any prereqs and perhaps more explanation through each step.

Reply:
I understand what you're saying, but I'm struggling to write this.

Reply:
Understandable. To start, we could add a sentence after the header that says like `Once you've [installed Meeshkan](#installation), you can execute the following commands using the Meeshkan CLI. This will help you get up-and-running with your very first Meeshkan project!` (or some variation of this)

We could break the commands up into sections (recorder, build, mock) and add one or two sentences before to briefly explain what is happening and where we're getting the data. 

Wdyt?